proxy/src/main/java/com/velocitypowered/proxy/connection/backend/BackendPlaySessionHandler.java

```diff
@@ -213,6 +216,11 @@ public boolean handle(ResourcePackRequest packet) {
     return true;
   }
 
+  @Override
+  public boolean handle(RemoveResourcePack packet) {
+    //TODO
```

Comment:
Shouln't this be done before a merge?

Reply:
Obviously, this pull request is still under development, it is not necessary to mention the remaining TODOs